x-pack/plugins/monitoring/public/components/cluster/listing/listing.js

```diff
@@ -21,16 +21,31 @@ import {
   EuiIcon,
   EuiToolTip,
 } from '@elastic/eui';
-import { EuiMonitoringTable } from '../../table';
+
 import { FormattedMessage } from '@kbn/i18n-react';
 import { i18n } from '@kbn/i18n';
+
+import { Legacy } from '../../../legacy_shims';
+import { EuiMonitoringTable } from '../../table';
 import { AlertsStatus } from '../../../alerts/status';
 import { STANDALONE_CLUSTER_CLUSTER_UUID } from '../../../../common/constants';
 import { getSafeForExternalLink } from '../../../lib/get_safe_for_external_link';
-import './listing.scss';
 import { useKibana } from '@kbn/kibana-react-plugin/public';
 import { toMountPoint } from '@kbn/react-kibana-mount';
 
+const clusterCellExpired = (theme) => css`
```

Comment:
Good point! I've used `someComponentStyle ` in majority. I don't mind changing all to `someComponentStyles` if that convention works better for you, although personally I wouldn't worry about such details because it's not a quick search and replace. Semantically "style" is correct because it does not imply a single style instruction but a general look.

Reply:
I've mainly been going based on what we use in EUI, which is `styles`. Kibana might have their own conventions.
So my main ask here is to make sure to use 1 naming pattern.

Reply:
@mgadewoll Ah, I gotcha! It seems in Kibana there's no one convention, there's even `componentCss`,`componentCSS` 😅 

I updated the cases in this file.